src/Rules/String.php

```diff
@@ -4,7 +4,7 @@
 
 use Rakit\Validation\Rule;
 
-class Alpha extends Rule
+class String extends Rule
```

Comment:
In PHP, case is often not important, so "String" cannot be used for the class name. It would be possible to write "StringRule", but this is not customary inside the library. I suggest renaming the class to "Text" and renaming the rule to "text". Maybe something doesn't sound right, I used a translator.

Reply:
Thanks Chamber6821.

That will teach me for doing too many things at once, late at night :stuck_out_tongue: